package.json

```diff
@@ -8,9 +8,7 @@
   "scripts": {
     "lint": "eslint .",
     "oyente": "oyente -s src/Fund.sol",
-    "devchain:before-tests": "parity --chain tests/parity-genesis.config --jsonrpc-apis all",
-    "before-tests": "babel-node tests/before-tests.js",
```

Comment:
think above two lines are still needed.
for [setup](https://github.com/melonproject/protocol/tree/develop#setup) instructions in readme

Reply:
Updating the readme